azure-iot-device/tests/iothub/pipeline/conftest.py

```diff
@@ -0,0 +1,7 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+pytest_plugins = ["tests.common.pipeline.fixtures"]
```

Comment:
As mentioned before, this throws warnings

Reply:
OK.  Will change.

---
In reply to: [291683903](https://github.com/Azure/azure-iot-sdk-python-preview/pull/104#discussion_r291683903) [](ancestors = 291683903)